### PR TITLE
Update Remix starter template

### PR DESCRIPTION
**Pull request by JackDevAU**
- Revamp of the whole template 
- Updates to latest version of remix (version 2.x.x) 
- Update latest to TinaCMS 

**Figure: New Remix project + TinaCMS**

Related issue - https://github.com/tinacms/tinacms/issues/5196

### CODE REVIEW

**Comment by Ben0189** [comment]
Hey @JackDevAU , can you link to an issue/PBI for this PR?

**Review by Ben0189 — APPROVED** [review]
LGTM

File: .github/workflows /pr-open.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: Build Pull request
+on:
+  pull_request:
```
Comment (Ben0189) [inline .github/workflows /pr-open.yml]: looks like you need to mention branches, to test if the action is working?

```yml 
    branches:
      - main
      - remix-starter-update
```

**Review by Ben0189 — CHANGES_REQUESTED** [review]

File: .github/workflows /pr-open.yml (hunk 1 above)
Comment (JackDevAU) [reply .github/workflows /pr-open.yml]: Seems its decided to work! :O

Comment (JackDevAU) [reply .github/workflows /pr-open.yml]: Removed as no longer needed ✅

**Review by Ben0189 — APPROVED** [review]

**Pull request merged**